SmartScrollView.js

```diff
@@ -1,12 +1,15 @@
 import React, {
   Component,
+  PropTypes,
+} from 'react';
+
+import {
```

Comment:
Could you also
`import ReactNative, {`

and update https://github.com/samin/react-native-smart-scroll-view/blob/eb44f6ddb63bb8c6fa5851aaab6d9b50041919f1/SmartScrollView.js#L123 from `React.findNodeHandle(this._smartScroll);` to `ReactNative.findNodeHandle(this._smartScroll);`